server/s3_get_object_action.h

```diff
@@ -32,6 +32,27 @@
 #include "s3_factory.h"
 #include "s3_timer.h"
 
+struct S3ExtendedObjectInfo {
+ public:
+  S3ExtendedObjectInfo() {
+    start_offset_in_object = 0;
+    object_OID = {0, 0};
+    object_layout = -1;
+    object_pvid = {0, 0};
```

Comment:
It can slash with my changes which are already merged to main branch.
Object pvid is already handled